### PR TITLE
Allow dependabot to check GitHub actions monthly

### PR DESCRIPTION
**Pull request by jlosito**
Let Dependabot update GitHub Actions dependency once a month.

Here's reference to the dependabot configs.

https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/keeping-your-actions-up-to-date-with-github-dependabot

Automerge-Triggered-By: GH:Mariatta

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@jlosito

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (Mariatta) [inline .github/dependabot.yml]: Can this be done monthly?

**Review by Mariatta — CHANGES_REQUESTED** [review]
Thanks. I think daily update is way too noisy for CPython. Can you change it to monthly?

**Comment by jlosito** [comment]
I have made the requested changes; please review again

**Review by Mariatta — APPROVED** [review]
Thanks!

**Comment by Mariatta** [comment]
Thanks for your first contribution to CPython! 🌮

**Comment by Mariatta** [comment]
I'm curious whether dependabot checks for the configuration on the other branches, or is it only for the main branch. I was thinking perhaps we should backport this change to 3.9 and 3.8, so at least we don't need to manually trigger backports to future dependabot PRs.

**Comment by hugovk** [comment]
It should be possible using the `target_branch` option:

https://dependabot.com/docs/config-file/#target_branch

In one of my repos, I have dependabot configured for `master`, and also for this PR's branch: https://github.com/hugovk/pypistats/pull/88

(I'm waiting for httpx to reach 1.0, and dependabot is updating dependencies to check it still passes CI in the meanwhile. It's been useful as the httpx mocker library has been out of sync a couple of times.)

I don't have config files for that repo yet, it's using config from the UI. The only difference between the two configs is there's a target branch set for the PR's branch.

**Pull request merged**